README.md

```diff
@@ -56,17 +56,43 @@ We currently target OpenFPGA for:
 ## Running with pre-built docker image
 
 ```bash
-# To get the docker image from the repository, 
+# To get the docker image from the repository,
 docker pull ghcr.io/lnis-uofu/openfpga-master:latest
 
 # To invoke openfpga_shell
 docker run -it ghcr.io/lnis-uofu/openfpga-master:latest openfpga/openfpga bash
 ```
 
+## Backward compatibility with OpenFPGA v1.1
+
+OpenFPGA v1.2 is a major upgrade over v1.1, which upgrades the internal VPR engine.
```

Comment:
Actually ``v1.2.0`` is the last release before vtr-upgrade. ``v1.2.x`` are based on vtr-upgrade

 If we want to avoid confusion, we just change the release name from 1.2.0 to 1.1.525. There is no difference between the v1.2.0 and v1.1.515(Final v1.1),  except the version number.  See if this makes sense.